ocamltest/main.ml

```diff
@@ -204,8 +201,6 @@ let test_file test_filename =
        if not !Options.log_to_stderr then close_out log;
        summary
     ) in
-  (* Restore current working directory  *)
-  Sys.chdir cwd;
```

Comment:
I tried to reconstruct the reasoning that this is safe. I believe it goes as follows: some parts of ocamltest do run `chdir` (the `cd` action and the `setup_*` actions), but in this piece of code the only place where they can run is guarded within a `with_chdir` wrappers that correctly resets the current directory.

Reply:
That's exactly it. In fact it was me that introduced the `with_chdir` at the time and I remember that I realised at the time that this `chdir` could go, but somehow I didn't get around to doing it.